### PR TITLE
bugfix: fix limit-count plugin redis.ttl error.

### LINKED ISSUE #1901: bug: 500 error when bench limit-count plugin
limit count conf:
policy: redis
count: 100
time_window: 60
key: remote_addr
os: CentOS 7.6
benchmark tool: apache benchmark

I got runtime error which result in 500 response when i bench 1million request, 1k concurrency by ab.It seems `red:ttl(key)` return nil in limit-count-redis.lua. The follow snippet is runtime error detail:
```
lua entry thread aborted: runtime error: .../apisix/apisix/plugins/limit-count/limit-count-redis.lua:76: attempt to compare nil with number
stack traceback:
coroutine 0:
        .../apisix/apisix/plugins/limit-count/limit-count-redis.lua: in function 'incoming'
        /usr/local/apisix/apisix/plugins/limit-count.lua:121: in function 'phase_fun'
        /usr/local/apisix/apisix/init.lua:147: in function 'run_plugin'
        /usr/local/apisix/apisix/init.lua:385: in function 'http_access_phase'
        access_by_lua(nginx.conf:161):2: in main chunk
```

Is it a bug? @membphis @moonming

### PR DESCRIPTION
**Pull request by shuaijinchao**
### What this PR does / why we need it:
FIX #1901 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Is this PR backward compatible?

### CODE REVIEW

File: apisix/plugins/limit-count/limit-count-redis.lua [hunk 1]
```diff
@@ -71,7 +71,8 @@ function _M.incoming(self, key)
     local remaining
     key = self.plugin_name .. tostring(key)
 
-    local ret = red:ttl(key)
+    -- todo: test case
+    local ret = red:ttl(key) or -2
```
Comment (membphis) [inline apisix/plugins/limit-count/limit-count-redis.lua:75]: I think we need to cache the `err`, eg: `local ret, err = red:(...)`

And we should write this error message to log, set the right response like `500`.

Comment (shuaijinchao) [reply apisix/plugins/limit-count/limit-count-redis.lua]: OK, fixed.

File: apisix/plugins/limit-count/limit-count-redis.lua [hunk 2]
```diff
@@ -72,7 +72,11 @@ function _M.incoming(self, key)
     key = self.plugin_name .. tostring(key)
 
     -- todo: test case
-    local ret = red:ttl(key) or -2
+    local ret, err = red:ttl(key)
+    if not ret then
+        return false, "failed to get redis ttl: " .. err
```
Comment (membphis) [inline apisix/plugins/limit-count/limit-count-redis.lua:77]: we'd better contain the `key` in the error message, it is useful

Comment (shuaijinchao) [reply apisix/plugins/limit-count/limit-count-redis.lua:77]: OK, good idea.

**Comment by membphis** [comment]
merged, many thx @shuaijinchao

**Pull request merged**